From 92acf5bd67adfb312e7b63c7dae8906d90c76dba Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Mon, 30 Oct 2017 11:13:32 +0800
Subject: [PATCH 2706/5242] MLK-16726 ARM64: dts: fsl-imx8qm-lpddr4-arm2:
 remove lpuart0 cts/rts pin

commit  7f4411841a37e61e67aa6fa944b3a3d840b9317e from
https://source.codeaurora.org/external/imx/linux-imx.git

Since lpuart0 configure as console port only, cts/rts pins are not necessary.
And lpuart2 is moved into M4 partition that will use these two pins.
So remove the two pins from dts file.

BuildInfo:
 - SCFW 88456c73, IMX-MKIMAGE 06bc2767, ATF a438801
 - U-Boot 2017.03-imx_v2017.03_4.9.51_imx8_beta1+g7953d47

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Reviewed-by: Ye Li <ye.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index 760779b..9a7ee1e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -274,8 +274,6 @@
 			fsl,pins = <
 				SC_P_UART0_RX_DMA_UART0_RX		0x06000020
 				SC_P_UART0_TX_DMA_UART0_TX		0x06000020
-				SC_P_UART0_RTS_B_DMA_UART0_RTS_B	0x06000020
-				SC_P_UART0_CTS_B_DMA_UART0_CTS_B	0x06000020
 			>;
 		};
 
-- 
1.7.9.5

